src/ui/file-browser/list/FilesListTable.js

```diff
@@ -0,0 +1,190 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Col, Alert, Spinner, Icon, DropdownKebab } from 'patternfly-react';
+import { FormattedMessage } from 'react-intl';
+import { LinkMenuItem } from 'frontend-common-components';
+import { ROUTE_FILE_BROWSER } from 'app-init/router';
+
+class FilesListTable extends Component {
+  componentWillMount() {
+    this.props.onWillMount();
+  }
+
+  renderTableRows() {
+    const downloadLabel = <FormattedMessage id="fileBrowser.downloadFile" />;
+    const deleteLabel = <FormattedMessage id="app.delete" />;
+
+    const getProtectedFolder = (path) => {
+      switch (path) {
+        case 'public':
+          return 'false';
+        case 'protected':
+          return 'true';
+        default:
+          return this.props.pathInfo.protectedFolder;
+      }
+    };
+
+    const getQueryString = (path) => {
+      switch (path) {
```

Comment:
This code is equivalent, and I think it could be more readable:
```javascript
if (['protected', 'public'].includes(path)) {
  return null:
}
return path;
```